### PR TITLE
PubSub - MQTT

### PR DESCRIPTION
**Pull request by johannes-wolf**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -521,7 +521,8 @@ main()
         local CMAKELISTS_DIR="${ZSERIO_PROJECT_ROOT}/compiler/extensions/cpp/runtime"
         local CPP_BUILD_DIR="${ZSERIO_BUILD_DIR}/runtime_libs/cpp"
         local CMAKE_ARGS=(-DCMAKE_INSTALL_PREFIX="${ZSERIO_DISTR_DIR}/runtime_libs"
-                          -DZSERIO_RUNTIME_INCLUDE_INSPECTOR=ON)
+                          -DZSERIO_RUNTIME_INCLUDE_INSPECTOR=ON
+                          -DZSERIO_RUNTIME_INCLUDE_PUBSUB_MQTT=ON)
```
Comment (josephbirkner) [inline scripts/build.sh]: Why is this necessary here? Just for test cases?

**Review by josephbirkner — COMMENTED** [review]
Looks good, only nitpick is MQTT build flag in build.sh.

**Comment by josephbirkner** [comment]
Please resolve merge conflicts ⚡️

**Comment by johannes-wolf** [comment]
Rebased onto master.

**Review by josephbirkner — APPROVED** [review]
💯
